hack/local-up-cluster.sh

```diff
@@ -1188,9 +1188,22 @@ fi
 
 if [[ "${START_MODE}" != "kubeletonly" ]]; then
   if [[ "${START_MODE}" != "nokubeproxy" ]]; then
-    start_kubeproxy
+    ## TODO remove this check if/when kubelet is supported on darwin
+    # Detect the OS name/arch and display appropriate error.
+    case "$(uname)" in
```

Comment:
`uname -s`?

Reply:
Added -s. I was just not sure if that is really needed!